From d8e306117f2af4c0da59d1a0b8dc886f5e83162d Mon Sep 17 00:00:00 2001
Message-ID: <d8e306117f2af4c0da59d1a0b8dc886f5e83162d.1729453324.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1729453324.git.sharlatanus@gmail.com>
References: <cover.1729453324.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 20 Oct 2024 15:49:46 +0100
Subject: [PATCH 25/42] gnu: python-sunpy: Update to 6.0.3.

* gnu/packages/astronomy.scm (python-sunpy): Update to 6.0.3.
[native-inputs]: Add python-setuptools and python-wheel.

Change-Id: I9bb936987043151d8d06cab46943ed807f127c47
---
 gnu/packages/astronomy.scm | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 6e849efe8c..80a534d815 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -4185,13 +4185,13 @@ (define-public python-sgp4
 (define-public python-sunpy
   (package
     (name "python-sunpy")
-    (version "6.0.2")
+    (version "6.0.3")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "sunpy" version))
        (sha256
-        (base32 "0mzmq2ncqgq61c1maxwynrmzcyiafnlil5mx4vhy2cvdyacm8yc9"))))
+        (base32 "1v3bnl9dhs9ks7jhmwy10qq6a99v2xcpalmigb255fxi9pqcy8lj"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -4230,7 +4230,9 @@ (define-public python-sunpy
            python-pytest-mock
            python-pytest-mpl
            python-pytest-xdist
-           python-setuptools-scm))
+           python-setuptools
+           python-setuptools-scm
+           python-wheel))
     (propagated-inputs
      (list parfive
            python-asdf
-- 
2.46.0

